### ISSUE TITLE
review fixed cost

### ISSUE THREAD

**Issue by nobodxbodon** [gratipay/inside.gratipay.com#928]
Reticket from #878. Below is based on [budget spreadsheet](https://docs.google.com/spreadsheets/d/1p3DpF9ZLEsViBx0685FwJaYN1vVScKVUgXHb2zyqXZg/edit?ts=581fd232#gid=1), https://github.com/gratipay/inside.gratipay.com/issues/73#issuecomment-58228856,  http://inside.gratipay.com/appendices/tools. Please list other missing in it.

As I'm not familiar with many services below, or the past related decisions, any input is more than welcome.

| Vendor | Weekly | Monthly | Yearly | Usage | Potential Alternatives | Note | 
| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
|Paramount | $28.75 | $125.00 | | Co-working space in Ambridge: #831 | |
|Heroku| $21.85	|$95.00	| | to review in  #946 | cloud/server provider like Linode [reference](https://blog.jalada.co.uk/heroku-is-no-longer-the-hobbyists-friend/), Redhat's Openshift
|Travis	|**$0**	|**$0**| | CI | | **free as we are OS** |
|PagerDuty	|$0 |$0	| | Removed | |
|Freshdesk	|$8.74	|$38.00	| | https://github.com/gratipay/inside.gratipay.com/issues/878#issuecomment-263072375 "$19 per seat and we have two seats".| | May cut in half by #936
|iWantMyName	|$6.70	|	|$349.40| Domain provider |  [other providers](https://stackshare.io/stackups/iwantmyname-vs-godaddy-vs-namecheap)
|GitHub	|**$0**	|**$0**| | [have](https://github.com/gratipay/inside.gratipay.com/issues/878#issuecomment-263072375) private repos for security & logs | | [sponsored](https://github.com/gratipay/inside.gratipay.com/issues/928#issuecomment-268379777)
|Pipedrive	|$0	|$0	| |  removed
|Sentry	|**$0** |**$0** | | Logging | | **sponsored**
|Transifex	|$4.37	|$19.00	| | manage translations | | [ on legacy plan](https://github.com/gratipay/inside.gratipay.com/issues/928#issuecomment-268284897) |
|MaxCDN	|$3.74	|$9.95	|$75.95| [host](https://github.com/gratipay/inside.gratipay.com/issues/712#issuecomment-241302409) assets. and downloads.| | [on legacy plan](https://github.com/gratipay/inside.gratipay.com/issues/928#issuecomment-268284897) |
|DigitalOcean	|$2.30	|$10.00	| | @clone1018 and @rohitpaulk 's backups [here](https://github.com/gratipay/inside.gratipay.com/issues/932#issuecomment-264220084) |
|StartSSL	|$2.30	|	|$119.80| [SSL vendor](https://github.com/gratipay/inside.gratipay.com/issues/73#issuecomment-53454153) | |
|Typography.com	|$1.90	|	|$99.00| #5 webfonts for gratipay site |
|Papertrail	|$1.61	|$7.00	| | Logging |
|Librato	|$1.15	|$5.00	| | [Monitor Logging](https://github.com/gratipay/inside.gratipay.com/issues/157#issuecomment-81719903) |
|LastPass	|$0.97	|	|$50.40 | #585 |
|DNSimple	|$0.69	|$3.00| | https://github.com/gratipay/inside.gratipay.com/issues/585#issuecomment-211981240 |

**Comment by whit537** [comment]
@nobodxbodon What's your goal, to spend as little as possible? :-)

**Comment by nobodxbodon** [comment]
Saving cost would be a side effect of this reviewing, IMO. It's mainly to check (hopefully periodically) whether we make the most out of each cost, by checking the current usage and exploring alternatives. These costs are recuring and can be more easily overlooked overtime. 

Besides, we may also consider @kaguillera [suggestion](https://github.com/gratipay/inside.gratipay.com/issues/878#issuecomment-263382629) of using OS service providers and tip them.

Additionally, it would also save managing cost if any of the service is no longer needed, or if we can use one provider instead of two similar ones.

**Comment by whit537** [comment]
Cool. I just want to make sure that we're on the same page in terms of the principle of paying for what we use. Gratipay exists to solve the free rider problem in open source, and its important to me that we model non-free-riding with our own vendors. That puts us at odds with traditional ways of thinking about accounting ... but that's what we're about! :-)

The reasons you mention are valid. I think we should focus on:

1. getting data into our accounting system (what @kaguillera is working on in https://github.com/gratipay/finances/pull/31)
1. standing up a reporting app (https://github.com/gratipay/finances/issues/28)

Once we have real data and a common way to look at it, then we can circle back here to have the _real_ discussion, about specific financial decisions. Does that sound right to you?

**Comment by whit537** [comment]
> Besides, I don't quite see dependency on the accounting system or reporting app. Those may have higher priority but this will have immediate influence on the cash flow.

Fair enough.

Looking over the list, the ones that jump out at me as providing low value are Pipedrive (#917), Librato (https://github.com/gratipay/gratipay.com/issues/4203), and possibly Papertrail (though I think @rohitpaulk will disagree on that one).

**Comment by nobodxbodon** [comment]
I didn't comment on Pipedrive as marketing is one of our weakness and I hoped the tool would help out. If you think it doesn't quite help with the marketing stategry/methods, +1 with removing it. 

Still the items above together is about half of Travis's cost. Now I feel that I must have missed something about Travis. Are we using some features that the commercial version provides, but the free version doesn't? Or is there some special reason that doesn't qualify us from using the free version?

**Comment by whit537** [comment]
@nobodxbodon We depend heavily on Travis. We need to give back somehow. If we don't give back monetarily, we need to find another way. What would you propose?

**Comment by nobodxbodon** [comment]
As I mentioned above, here are just some ways I can think of. I'm sure there are many other ways.

- put Travis logo on the homepage stating that our project use it, together with other core/free services
- contributing to [Travis project on github](https://github.com/travis-ci/travis-ci) in manpower (discussion, coding, etc)
- sharing experience of using Travis in our project (twitter, tech articles, blog, etc)

**Comment by whit537** [comment]
Those are abstract possibilities, yes. Do you have a concrete proposal for what to replace our monetary payment with? Are you volunteering to do any of the things you mention? :-)

**Comment by nobodxbodon** [comment]
For No.1, how about something like below:
![screenshot from 2016-12-03 00 43 50](https://cloud.githubusercontent.com/assets/392497/20858116/c0d56594-b8f1-11e6-8185-b2bed13ea347.png)

**Comment by whit537** [comment]
I don't think that's equivalent to $69/mo. Do you?

**Comment by rohitpaulk** [comment]
I'm all for paying for what we use, but this must be value driven.

I don't think $69/mo for Travis is justified at all in the first place - shouldn't it be along the lines of $25 (how much we pay Github)? 

Heroku is the one item I definitely wouldn't move - $95/month is definitely worth it. We get database backups, monitoring, easy rollbacks + a reliable service overall. Trying to replicate that on another provider (like AWS) is likely going to cost us as much or more (including man hours).

**Comment by nobodxbodon** [comment]
> I don't think that's equivalent to $69/mo. Do you?

IMO it's potentially worth more if compared to ads.

> Heroku is the one item I definitely wouldn't move

I agree though there are [alternatives](http://www.topbestalternatives.com/heroku/) to Heroku, I haven't seen any has overall better score, considering feature, cost, maturaity, etc.

**Comment by JessaWitzel** [comment]
I think that paying Travis $70/month is better than dedicating man hours to it that should be dedicated here but I also think that Travis deliberately offers free accounts to open source projects because they have the same value system as *we* do and that (much like our technical debt discussion) it is useful to have a base goal before you prioritize idealism over bottom line. #imho

**Comment by nobodxbodon** [comment]
FYI, another service we may consider using is [Netlify](https://www.netlify.com/pricing/), as they also are free specially for OS projects:
![screen shot 2016-12-05 at 10 01 08 am](https://cloud.githubusercontent.com/assets/392497/20896298/f63f3dd6-bad1-11e6-8272-b239c73202ad.png)

**Comment by whit537** [comment]
> they have the same value system as _we_ do

Say more? What I'm seeing is that we have a [PWYW](https://en.wikipedia.org/wiki/Pay_what_you_want) pricing model (with a floor), and Travis has a [freemium](https://en.wikipedia.org/wiki/Freemium) model. 

I'm not comfortable being on the "free" side of a freemium business model. Somehow it's related to my even greater discomfort being on the "free" side of an [adware](https://en.wikipedia.org/wiki/Adware) business model. I don't know how best to articulate my discomfort right now (I've tried several times today and want to work on other things :). I mean, we've talked about this in years past, so there's almost certainly a record in the old IRC archives if not GitHub. That doesn't mean we don't need to hash it out again with a new Gratipay generation. :-)

Something blah blah, "[If you are not paying for it, you're not the customer; you're the product being sold](http://www.metafilter.com/95152/Userdriven-discontent#3256046)." Blah blah something. [But](https://www.techdirt.com/articles/20121219/18272921446/stop-saying-if-youre-not-paying-youre-product.shtml).

> I don't think $69/mo for Travis is justified at all in the first place - shouldn't it be along the lines of $25 (how much we pay Github)?

I'd be totally fine switching (paid) CI vendors. The next step there would be a comparison with one or more alternatives. Note that we do use a private project as part of our [security workflow](http://inside.gratipay.com/howto/handle-security-issues).

**Comment by nobodxbodon** [comment]
Docker can be alternative to Heroku. There will be some learning curve, but it's becoming trendy.

Additionally, Redhat offers [Resource Grants Program](https://www.openshift.com/grants/) for OS projects to run on their Openshift docker platform.

**Comment by Changaco** [comment]
> Redhat offers [Resource Grants Program](https://www.openshift.com/grants/) for OS projects to run on their Openshift docker platform.

Not really, OpenShift v3, the new version based on Docker, isn't actually launched yet, it's still in developer preview (temporary accounts, can't host real projects on it).

**Comment by nobodxbodon** [comment]
> Note that we do use a private project as part of our security workflow.

@whit537 is that a close source project or just a repository to postpone open sourcing the security fixes after they are fixed? If latter it should be still considered OS project I suppose?

> Not really, OpenShift v3, the new version based on Docker, isn't actually launched yet, it's still in developer preview

@Changaco thanks I overlooked [that](https://docs.openshift.com/online/getting_started/devpreview_faq.html). Still the [migration](https://docs.openshift.com/dedicated/dev_guide/migrating_applications/index.html) from v2 app to v3 seems pretty straightforward, and it can take a while to sign up for the Resource Grants Program. So maybe worthwhile to consider starting the process and putting some part of Gratipay on it.

**Comment by whit537** [comment]
> is that a close source project or just a repository to postpone open sourcing the security fixes after they are fixed? If latter it should be still considered OS project I suppose?

The latter. The question isn't whether we _could_ get free service from Travis, GitHub, etc. We [almost certainly could](https://twitter.com/nayafia/status/805937393312272384). The question is whether we _should_ as a matter of policy.

**Comment by mattbk** [comment]
@JessaWitzel pretty much captured my thoughts.

**Comment by JessaWitzel** [comment]
What I mean by same value system is that they obviously want to encourage open source work. We are trying to allow people to get paid for open source work and they are trying to support people who are doing open source work by offering them value for free. ["Travis CI is, and always will be, free for open source projects."](https://docs.travis-ci.com/user/travis-pro/#Why-can%E2%80%99t-I-find-information-on-pricing-on-travis-ci.org%3F) I don't see an ethical dilemma in taking advantage of the value that they want to offer us until we are bootstrapped (am I using that word right?). I do, however, see why you would want to follow the "you should contribute in some way to the open source code that you use" mantra and set a good example. I just think that Travis has intentionally set this pricing structure up because they want us to be able to take advantage of it, which makes it a little different? << The question mark is there because maybe it's not different.

**Comment by whit537** [comment]
Alright, so what's our deadline for paying Travis? If we stop paying them now, will we ever start paying them again? When? How will we know?

**Comment by nobodxbodon** [comment]
> If we stop paying them now, will we ever start paying them again? 

I suppose you mean we'll take the free services for granted and forget about them while using them. If we review services we use periodically (how about twice a year?), that will give good opportunity to decide whether we are in good position to start paying for certain free services.

> When? How will we know?

I don't see any other way than to decide case by case. Similarly, it's hard to tell how much is proper to give to a certain free service to show gratitude.

**Comment by mattbk** [comment]
What would we use $69/mo (~$17/wk) for, if it wasn't going to Travis? Would it be added to https://gratipay.com/Gratipay/distributing/? Would it be used for some other vendor? Would it be saved up for conferences/other travel to drum up business?

I think that question needs to be answered in order to determine whether money spent on Travis is worth more than money spent elsewhere.

Would Travis be willing to open a project on Gratipay so open-source projects (like ours) can give them an affordable amount, even though it isn't necessary per Travis' open-source-is-free rule? PWYW for the free part of a Freemium model?

**Comment by whit537** [comment]
> I think that question needs to be answered in order to determine whether money spent on Travis is worth more than money spent elsewhere.

If we're going to accept free services from Travis, it'll be because we're willing to accept free services in general, which means that our cost for GitHub, [Sentry](https://sentry.io/for/open-source/), and possibly others (Heroku? Transifex? Librato?) would go away as well.

> Would Travis be willing to open a project on Gratipay so open-source projects (like ours) can give them an affordable amount, even though it isn't necessary per Travis' open-source-is-free rule? PWYW for the free part of a Freemium model?

No. I've actually talked to [rkh](https://github.com/rkh) about it. They're off and running and it would be more hassle than it'd be worth to accept money PWYW.

**Comment by whit537** [comment]
> our cost for GitHub, Sentry, and possibly others (Heroku? Transifex? Librato?) would go away as well.

... unless there's a reason why we would keep paying some vendors that offer free services and not others? I suppose it could be because of case-by-case considerations: this vendors needs it more than this other. I think the way we know whether a vendor needs it is that they stop offering free services, or clearly communicate that they are PWYW. In fact, Read the Docs is a vendor that [falls](https://readthedocs.org/accounts/gold/subscription/) into this latter category. We should probably start paying them!

![screen shot 2016-12-07 at 2 01 08 pm](https://cloud.githubusercontent.com/assets/134455/20982231/a2eb516e-bc85-11e6-963f-ea808448fd47.png)

**Comment by nobodxbodon** [comment]
> I suppose it could be because of case-by-case considerations: this vendors needs it more than this other. I think the way we know whether a vendor needs it is that they stop offering free services, or clearly communicate that they are PWYW.

+1 

Besides, is the option of listing all free services on bottom of homepage on the table? Something like:

## `Gratifully powered by: Heroku    TravisCI    GitHub ...    `

**Comment by nobodxbodon** [comment]
> What would we use $69/mo (~$17/wk) for, if it wasn't going to Travis?

@mattbk IMO it should be part of budget planning, which I don't think there's a formal one for now? Like what we have, how much is spending, how much to spend and how & when, etc. 

Anyways, as I mentioned IMO there are many possibilities to spend that in a more direct way to help Gratipay grow.

**Comment by whit537** [comment]
> Besides, is the option of listing all free services on bottom of homepage on the table?

How about listing *all* services that we use?

**Comment by nobodxbodon** [comment]
> How about listing all services that we use?

+1. How about highlighting those that are only free because of our open source nature?

**Comment by mattbk** [comment]
How about [leveling](https://github.com/gratipay/gratipay.com/issues/3736#issuecomment-248949250) or [badging](https://github.com/gratipay/gratipay.com/issues/3963#issuecomment-265887061) external services the same way we end up doing for Gratipay projects?

**Comment by JessaWitzel** [comment]
I love all of this. FYI

**Comment by nobodxbodon** [comment]
Any action items for this?

**Comment by whit537** [comment]
I think we're ready to move forward with applying for free service from various vendors. I've started with Sentry.

![screen shot 2016-12-19 at 11 37 45 am](https://cloud.githubusercontent.com/assets/134455/21320591/9ab2d4dc-c5df-11e6-803f-424e65717f43.png)


> Besides, is the option of listing all free services on bottom of homepage on the table?

I don't like listing free services but not paid services, and I'd rather not clutter up the bottom of the homepage if we don't have to. I think the bigger picture here is that we need to develop an internal capacity to be aware of the vendors we're depending on, and make sure we're giving back to them one way or another. Some services may specifically request that we give them a link on our homepage. Let's see what happens when we get into it here ...

**Comment by whit537** [comment]
I've canceled our Travis subscription, and emailed Nadia at GitHub to ask about [free private repos](https://twitter.com/nayafia/status/805937393312272384).

**Comment by whit537** [comment]
> We offer Transifex for free to Open Source projects that have no funding, revenue, and/or commercialization model. If you have a question whether your organization qualifies for our Open Source program, please contact us.

https://www.transifex.com/pricing/

Should we contact them?

**Comment by whit537** [comment]
Should we drop PagerDuty? We already have Uptime Robot—that's where Pager Duty gets its events from. If we drop PagerDuty then we only have email notifications instead of text messages. Can we live with that, @clone1018 @rohitpaulk?

**Comment by mattbk** [comment]
I think it would be stretching to stop paying for Transifex under that definition.

**Comment by whit537** [comment]
We can buy SMS messaging from Uptime Robot according to this schedule:

| n | $ | $ / ea |
|:----|:-----|:-------|
| 50 | 15 | 0.30 |
| 100 | 25 | 0.25 |
| 250 | 55 | 0.22 |
| 500 | 100 | 0.20 |

I am looking at the logs in Uptime Robot to get a sense of how fast we would use messages (PagerDuty was sending us a lot there for a while, though I think we reconfigured to back off?). If they only send one message per event then 50 messages would last us a while, even if we're sending three messages per event (me, @rohitpaulk, and @clone1018).

```
Event,Date-Time,Reason,Duration,"Duration (in mins.)"
Up,"2016-11-19 19:14:14",OK,"717 hrs, 53 mins",43074
Down,"2016-11-19 18:40:53","Service Unavailable","0 hrs, 33 mins",33
Up,"2016-09-28 08:45:34",OK,"1257 hrs, 55 mins",75475
Down,"2016-09-28 08:35:44","Connection Timeout","0 hrs, 9 mins",10
Up,"2016-09-28 08:30:58",OK,"0 hrs, 4 mins",5
Down,"2016-09-28 08:29:44","Connection Timeout","0 hrs, 1 mins",1
Up,"2016-09-28 08:24:43",OK,"0 hrs, 5 mins",5
Down,"2016-09-28 08:23:43","Connection Timeout","0 hrs, 1 mins",1
Up,"2016-09-28 08:18:43",OK,"0 hrs, 5 mins",5
Down,"2016-09-28 07:52:53","Connection Timeout","0 hrs, 25 mins",26
Up,"2016-09-28 07:48:02",OK,"0 hrs, 4 mins",5
Down,"2016-09-28 07:47:08","Service Unavailable","0 hrs, 0 mins",1
Up,"2016-09-04 07:54:38",OK,"575 hrs, 52 mins",34553
Down,"2016-09-04 06:17:16","Service Unavailable","1 hrs, 37 mins",97
Up,"2016-07-20 10:20:02",OK,"1099 hrs, 57 mins",65997
Down,"2016-07-20 10:17:42","Connection Timeout","0 hrs, 2 mins",2
Up,"2016-07-07 11:11:47",OK,"311 hrs, 5 mins",18666
```

We can also use carrier-supplied email-to-SMS for a number of carriers, including Idea Cellular in India, and dozens of options in the USA. Their disclaimer:

> **Important!** Uptime Robot uses free e-mail to SMS gateways of the providers that support it. These gateways are not always stable and such free SMS notifications may not always work as expected.

PagerDuty is pretty expensive for how much we're using it. I propose we turn off PagerDuty and buy a block of 50 SMS messages at Uptime Robot and see how we get along with that. Eh?

**Comment by clone1018** [comment]
I think that's a great idea.

**Comment by whit537** [comment]
Gosh. I can't find that PagerDuty is even charging us. On their website I find one invoice from 2014, and I don't find anything in PNC in the past couple months. 😳

**Comment by clone1018** [comment]
Last time I got a PagerDuty alert was 7/7/14

**Comment by whit537** [comment]
I think that was because you were off the rotation, though. @rohitpaulk and I have definitely gotten them more recently.

**Comment by whit537** [comment]
Grepping our [bank logs](https://github.com/gratipay/logs/tree/master/statements) only turns up that one charge, back in 2014. Sorry, PagerDuty? 😕

**Comment by whit537** [comment]
Okay! Well, let's give @rohitpaulk a day or three to respond, but I think it looks like we're going to whack PagerDuty and go with SMS directly from Uptime Robot.

**Comment by mattbk** [comment]
>We're sorry to see you go!
Your subscription has been cancelled.
This email is to confirm your subscription with Travis CI has been cancelled and is set to expire on December 29, 2016.
If you cancelled by mistake, please respond to this email, and we'll help you getting back up and running.
Thank you,
The Travis CI Team
https://gratipay.freshdesk.com/helpdesk/tickets/6411

**Comment by whit537** [comment]
I've removed Pipedrive ($25.00) and Travis ($70.93) from the [budget](https://docs.google.com/spreadsheets/d/1p3DpF9ZLEsViBx0685FwJaYN1vVScKVUgXHb2zyqXZg/edit?usp=sharing), which already takes us from $127.40/wk = $553.96/mo = $6,647.51/yr down to $105.34/wk = $458.03/mo = $5,496.35/yr. That's a savings of 17.3%! Let's budget $25/yr for Uptime Robot. Since apparently we haven't been paying PagerDuty for some reason, that's a slight increase. Also, we never adjusted the budget when @gratipay-bot came online, so that's a slight increase from $5/mo to $10/mo. With those adjustments, if Sentry and GitHub are kind to us then we will land at $85.53/wk = $371.91/mo = $4,462.95/yr, or two-thirds of where we were before this exercise.

!m @nobodxbodon *

**Comment by whit537** [comment]
> Hey Gratipay, 
>
We’ve just upgraded your account (Gratipay) to our Sponsored plan. If you have any other questions let us know. 
>
Thanks for choosing Sentry! 
>
p.s. We’d appreciate if you’d tell your friends, or maybe toss up a Sentry link in your app to help us spread the word! You can always find high quality copies of our logo on our Branding page (https://sentry.io/branding/).

**Comment by clone1018** [comment]
What is `Paramount`?

**Comment by whit537** [comment]
@clone1018 Co-working space in Ambridge: https://github.com/gratipay/inside.gratipay.com/issues/831. Open to discussing ...

**Comment by clone1018** [comment]
@whit537 You're Gratipay's most valuable asset. So if you use Paramount and you find value in it, I approve!

**Comment by whit537** [comment]
@kaguillera @dmk246 and I work there together on Wednesdays, and at my place on Thursdays. We could cut out Paramount and just work at my place both days if we wanted to save that money. To be honest, Paramount is about twice as expensive as other coworking spaces down in Pittsburgh—and it's not down in Pittsburgh, it's way out here in Ambridge. We're their first customers, and I kinda wanna encourage them since they're trying to Do Something&trade; in Ambridge. We could use their space if we have the retreat (#927) in Ambridge. I'm a bit torn.

**Comment by nobodxbodon** [comment]
Great moves!

> I don't like listing free services but not paid services, and I'd rather not clutter up the bottom of the homepage if we don't have to. 

+1.

> I think the bigger picture here is that we need to develop an internal capacity to be aware of the vendors we're depending on, and make sure we're giving back to them one way or another. 

IMO there'll be benefits if we shout louder about the vendors. As this internal auditing already gets us something in return, I can't really imagine what (constant) public review would bring us.

> Some services may specifically request that we give them a link on our homepage. Let's see what happens when we get into it here ...

Though giving link on homepage is just one option of giving back, It does have little maintenance cost and high visibility, which may be part of the reason why many startups have a section of big-name users on their homepages. In our case, other than the benefits above (constant public review), IMO it'll also give us another "model" role in the OS community, which shows how exactly an open company can run technically, especially with help from those vendors who perticularly support OS. 

As I mentioned in other tickets, I think we already are open enough in nature, but not enough people are aware of that.

**Comment by nobodxbodon** [comment]
> We could cut out Paramount and just work at my place both days if we wanted to save that money. 

Based on my experience, it makes a big difference between WFH and having a workspace, which was why I didn't comment on it in the first place.

> To be honest, Paramount is about twice as expensive as other coworking spaces down in Pittsburgh—and it's not down in Pittsburgh, it's way out here in Ambridge.

If only consider this, assuming Ambridge isn't close to any of you, it's a no-brainer in my view to switch.

> We're their first customers, and I kinda wanna encourage them since they're trying to Do Something™ in Ambridge.

IMO it doesn't hurt to negotiate with them, with the examples of other vendors waiving our cost or sponsoring, because of our open nature.

**Comment by nobodxbodon** [comment]
@whit537 I updated the budget table on the top. Could you check the cost of Transifex and MaxCDN, as I didn't find corresponding plans or their sites.

**Comment by nobodxbodon** [comment]
Would it be worthwhile to create another/sub ticket to discuss the hosting service? Heroku may seem to be the obvious choice, but there are alternatives out there. For sure it will be the most tech-savvy mission to switch among all vendors, if we decide to, but at least we can start from finding all the benefits from Heroku which can not be easily replaced by alternatives.

**Comment by whit537** [comment]
> Great moves!

Thanks! 💃

> Could you check the cost of Transifex and MaxCDN, as I didn't find corresponding plans on their sites. 

We're on legacy plans with Travis and MaxCDN. Same with DNSimple, iirc, as well as PagerDuty and GitHub.

Re: GitHub ... I heard back from Nadia and we have the green light! 💃

What should we pay RtD? $25/mo?

**Comment by whit537** [comment]
I started thinking about migrating away from Heroku, and quickly felt like it was something I didn't care to do. So much work! If you want to reticket, go for it.

**Comment by whit537** [comment]
> assuming Ambridge isn't close to any of you

Ambridge is [very close to me](https://gratipay.news/meet-ambridge-8b9b2b6f8628). :-)

**Comment by nobodxbodon** [comment]
> Ambridge is very close to me. :-)

Makes total sense to me now. Thanks for the background. Scratch what I said about switching or negotiating above please.

**Comment by nobodxbodon** [comment]
> What should we pay RtD? $25/mo?

What's RtD?

**Comment by whit537** [comment]
Read the docs. Used for:

http://gratipay.readthedocs.io/
http://environment-py.readthedocs.io/en/latest/
https://postgres-py.readthedocs.io/en/latest/

**Comment by whit537** [comment]
> Scratch what I said about switching or negotiating above please.

No worries. @kaguillera and I also co-work at [Catapult](http://catapultpgh.org/) (a Gratipay [customer](https://gratipay.com/catapultpgh/)! 💃 ) and [Stack](http://www.stackpgh.com/), but we pay for those personally.

**Comment by whit537** [comment]
And one thing I _would_ be comfortable negotiating would be letting us _all_ use the space during the retreat (#927) if we have it in Ambridge.

**Comment by whit537** [comment]
Over on https://github.com/gratipay/inside.gratipay.com/issues/137 I'm starting to feel how valuable accepting gifts is to building relationships with other companies, and how valuable that can be to our reputation and standing in the community. I want to take to Twitter and thank Sentry (I heard separately from the founder, David Cramer) and Sticker Mule and Read the Docs and ...

I am starting to warm to the idea of doing the same at the bottom of our homepage.

**Comment by whit537** [comment]
Nadia: "Great! I just applied it to the Gratipay org so you get free private repos forever. :)"

💃 

Coordinating w/ her re: Tweet a la https://github.com/gratipay/inside.gratipay.com/issues/137#issuecomment-268377437 ...

**Comment by whit537** [comment]
💃

![screen shot 2016-12-20 at 5 54 55 pm](https://cloud.githubusercontent.com/assets/134455/21371139/82b1756a-c6dd-11e6-98d4-954e3ef2effe.png)

**Comment by whit537** [comment]
> I am starting to warm to the idea of doing the same at the bottom of our homepage.

But let's not forget that if we do that, then we should do so as part of promoting people and organizations that back us financially on our own platform (cf. https://github.com/gratipay/gratipay.com/issues/236).

**Comment by nobodxbodon** [comment]
FYI I'm trying to fill in "usage" column of each vendor on top, and hopefully reference history ticket to add some background, like why we chose that vendor in the first place.

**Comment by nobodxbodon** [comment]
According to #44, are we not using MaxCDN now?

**Comment by whit537** [comment]
@nobodxbodon We are definitely using MaxCDN. #44 is for http://inside.gratipay.com/. We use MaxCDN to serve https://assets.gratipay.com/, which is used by https://gratipay.com/.

**Comment by whit537** [comment]
> FYI I'm trying to fill in "usage" column of each vendor on top, and hopefully reference history ticket to add some background, like why we chose that vendor in the first place.

Okay, make sure you update the spreadsheet and not just this ticket, since we'll be closing this soon I think. :-)

**Comment by EdOverflow** [comment]
Would it be possible to add [Cloudflare](https://www.cloudflare.com/) to the discussion? Here is an article by Troy Hunt on how and why he uses Cloudflare: https://www.troyhunt.com/cloudflare-ssl-and-unhealthy-security-absolutism/

**Comment by whit537** [comment]
@EdOverflow Yes, but let's make a new ticket for that please. This one is getting crowded and should be brought in for a landing.

**Comment by nobodxbodon** [comment]
Just one more question, and then the only left piece will be #946.

Is Papertrail used for raw log storage and Sentry for event notification and debugging? So the functions are not overlapped?

**Comment by whit537** [comment]
> Is Papertrail used for raw log storage and Sentry for event notification and debugging? So the functions are not overlapped?

Correct.

**Comment by whit537** [comment]
We also need to decide how much we're going to pay Read the Docs, and [now](https://github.com/gratipay/gratipay.com/pull/4244) Report URI as well.

**Comment by EdOverflow** [comment]
@whit537 I just wanted to add that I promised [Scott Helme](https://scotthelme.co.uk/) (the creator of Report URI) that I would write a blog post about how we use his security tools, which he could use to promote his products. :)

**Comment by whit537** [comment]
@EdOverflow Awesome. Can we publish that on https://gratipay.news/? :-)

**Comment by EdOverflow** [comment]
@whit537 Of course!

**Comment by nobodxbodon** [comment]
@whit537 About RtD, in their [statement](https://readthedocs.org/accounts/gold/subscription/) in my account it says:

![screen shot 2017-01-02 at 1 50 02 pm](https://cloud.githubusercontent.com/assets/392497/21596638/76fefa16-d0f2-11e6-9456-e827eeaca2b8.png)

IMO the basic commercial plan ($50/month) is beyond our needs (15 users included
50000 monthly pageviews included). How about paying the suggested least $20/month, plus put them in "Powered by" part in #950?

About Report URI, I haven't seen any suggestion about the amount of paying/pricing or the estimation of their spending on the site, so I don't have much of opinion yet. Maybe @EdOverflow has better idea?

**Comment by nobodxbodon** [comment]
@whit537 another thought about RtD. I noticed they have sponsor & supporter list [on sustainability page](https://readthedocs.org/sustainability/). Also they mention their top sponsor Rackspace on homepage. Though we may not in the position be their top sponsor at this moment, but still it can be something to consider in the future.

BTW I see you in their supporter list. I'm wondering how they choose to put company name or person name.

**Comment by whit537** [comment]
I paid personally, that's why I'm on there personally.

**Comment by whit537** [comment]
Travis customer success reached out in private email regarding our cancellation, and I am in a conversation with them about possibly sponsoring us for a private build for our security repo.

**Comment by whit537** [comment]
I was reminded under https://github.com/gratipay/gratipay.com/pull/4232 that we also uses Amazon Web Services, currently just for sending email. I believe we are still under their free tier. I'm not sure how long that will last.

**Comment by whit537** [comment]
I've added AWS to the [spreadsheet](https://docs.google.com/spreadsheets/d/1p3DpF9ZLEsViBx0685FwJaYN1vVScKVUgXHb2zyqXZg/edit#gid=1).

**Comment by whit537** [comment]
Good news from Travis!

> We would be happy to give you a free 1 concurrent build subscription (i.e. Bootstrap plan) for help you with all the good you are doing for open source. In return, do you think you could add a mention about our sponsorship on your website or write a blog post? How does this sound?

💃 

My response:

> Ah, thanks so much, []! That is wonderful news! :D
>
>We would love to feature Travis in a blog post alongside GitHub and Sentry, as well as to acknowledge your generosity on Twitter. Does that sound good? What would next steps be?

**Comment by whit537** [comment]
> alongside GitHub and Sentry

And Sticker Mule. :-) Anyone else I'm forgetting?

**Comment by nobodxbodon** [comment]
Speaking of which, do you think https://github.com/gratipay/inside.gratipay.com/issues/950 would be a more persistent way to show the acknowledgement from these vendors?

**Comment by nobodxbodon** [comment]
Is https://github.com/gratipay/inside.gratipay.com/issues/333 still relevent?

**Comment by whit537** [comment]
Yes, more persistent, but we're unresolved there due to the question of also acknowledging the people that give money to Gratipay and in some cases have done so for years. Since Travis offered a blog post that seemed the easiest way forward.

**Comment by whit537** [comment]
> Is #333 still relevent?

Yes, we still use Icomoon. Good catch.

**Comment by whit537** [comment]
> Yes, we still use Icomoon.

I've added them to the [budget](https://docs.google.com/spreadsheets/d/1p3DpF9ZLEsViBx0685FwJaYN1vVScKVUgXHb2zyqXZg/edit) (btw, once https://github.com/gratipay/finances/issues/12 is done, budget changes will be by PR in the finances repo, and will be much more auditable).

Apart from settling with Travis, I believe our last task here is to decide on how much we're paying the donation-based services we depend on, and then we can write up a blog post and close this ticket. Does that sound right to you, @nobodxbodon et al.?

**Comment by mattbk** [comment]
> budget changes will be by PR in the finances repo, and will be much more auditable).

This makes me happy.

**Comment by whit537** [comment]
From Travis:

> Thank you for your positive reply as well! :)
>
>Please send us links to your blog post and tweet and we will be happy to setup the free 1-build subscription for a year. In the meantime, I've given the Gratipay organization 25 trial builds if you need them.
>
>I hope this works for you and we are eager to read that blog post!
>
>Wishing you a great start into the week.

**Comment by nobodxbodon** [comment]
As there's discussion in slack about using mockup service, here's another option: https://balsamiq.com/free/ under category "I contribute to a big Open Source project"

**Comment by nobodxbodon** [comment]
Applied for free mybalsamiq license:

> Dear sir or madam,

> the Gratipay project aims to help people sustain open-source projects. I'm proud to take part in it, and as we notice there are a lot to improve in UX and I believe Balsamiq would help us express and communicate our thoughts better, and minimize rework caused by suboptimal designs.

> Here's our activity status: https://www.openhub.net/p/gratipay
> Third party introduction: https://opensource.com/article/16/12/how-gratipay-solves-open-sources-free-rider-problem

> My github account is https://github.com/nobodxbodon

> Thanks for consideration.

**Comment by nobodxbodon** [comment]
@whit537 mybalsamiq has agreed to provide *Open Source free forever* setting for the site (place to create mockup projects) I created. Could you give an email so that I can invite you as staff and then transfer the site to you?

**Comment by whit537** [comment]
Let's use vendors@gratipay.com. Thanks, @nobodxbodon!

**Comment by nobodxbodon** [comment]
@whit537 invitation sent. Could you check please?

**Comment by whit537** [comment]
Got it! Any chance we can change from https://gratipay-ux.mybalsamiq.com/ to just https://gratipay.mybalsamiq.com/?

**Comment by nobodxbodon** [comment]
Owner transferred. As I recall I saw in config a place to change the site name, as far as it's not taken I guess.

**Comment by whit537** [comment]
Awesome, changed!

!m @nobodxbodon

**Comment by whit537** [comment]
I've added Balsamiq to the [spreadsheet](https://docs.google.com/spreadsheets/d/1p3DpF9ZLEsViBx0685FwJaYN1vVScKVUgXHb2zyqXZg/edit#gid=1).

**Comment by whit537** [comment]
$20/mo to Read the Docs!
https://twitter.com/Gratipay/status/839964593246130177

**Comment by whit537** [comment]
>>We should probably take the $20/mo we'll save at Heroku and donate it to Let's Encrypt.
>>A dedicated SSL certificate on Cloudflare costs 5$ a month.
>Sooooo ... $10/mo donation?

Our budget for StartSSL had been $119.80/yr, sooooo ...

$10/mo to Let's Encrypt!
https://twitter.com/Gratipay/status/839969939188490241

**Comment by whit537** [comment]
Upon further [inspection](https://icomoon.io/#premium), IcoMoon is actually a freemium model, not a pwyw model. We're fine at the free level there, I think.

**Comment by whit537** [comment]
Report URI ... figure $5/mo, ya? No easy recurring, soooo ...

$60 to Report URI!
https://twitter.com/Gratipay/status/839969939188490241

**Comment by whit537** [comment]
>  I believe our last task here is to decide on how much we're paying the donation-based services we depend on, and then we can write up a blog post and close this ticket.

Okay! Pwyw services are topped up! Let's use https://github.com/gratipay/inside.gratipay.com/issues/950 for the blog post. Closing! 💃 🌻 

!m @nobodxbodon 
!m *
